src/bytes.js

```diff
@@ -1,18 +1,5 @@
 const empty = new Uint8Array(0)
 
-/**
- * @param {Uint8Array} d
- */
-const toHex = d => d.reduce((hex, byte) => hex + byte.toString(16).padStart(2, '0'), '')
-
-/**
- * @param {string} hex
- */
-const fromHex = hex => {
-  const hexes = hex.match(/../g)
-  return hexes ? new Uint8Array(hexes.map(b => parseInt(b, 16))) : empty
-}
-
```

Comment:
this is a breaking change, they're used pretty heavily outside of this package since you can get `bytes` from the main export and having simple hexification functions without having to reach for another package is quite handy